atom.mk

```diff
@@ -165,3 +171,15 @@ LOCAL_LIBRARIES := cryptsetup
 LOCAL_COPY_FILES := scripts/volmount/crypt/crypt:lib/pv/volmount/crypt/crypt
 
 include $(BUILD_CUSTOM)
+
+include $(CLEAR_VARS)
+
+LOCAL_DESTDIR := ./
+LOCAL_MODULE := pvzram
+
+LOCAL_C_INCLUDES := $(LOCAL_PATH)
+
+LOCAL_SRC_FILES := disk/disk_zram_utils.c \
+		   utils/pv_zram.c
+
+include $(BUILD_EXECUTABLE)
```

Comment:
what is this new executable for?

Reply:
This is a command line app to provisioning new zram disk. The idea is, for example, be able to add swap to a running device. This application use the same library used by the internal code.

I think I need to change this to be included in /usr/bin or any other directory in the PATH